### ISSUE TITLE
Feature request: mark as unread

### ISSUE THREAD

**Issue by oliverschwendener** [wireapp/wire#223]
Hello Wire team

It would be nice if I could mark a message/chat as unread.

Thank you, I really enjoy using wire!

**Comment by Filbuntu** [comment]
Hello Astrid!
Any progress on this?
Thanks!

**Comment by Filbuntu** [comment]
Hello Wire-team!
Any news regarding this feature. Recently I almost forgot an important message as I can not unread it. 
Thanks for working on this! It would be make such a difference for all users, also for the business users.

**Comment by Filbuntu** [comment]
Hello @Astrid-Wire and @lucilacaranza!
I would like to inquire regarding this as it is such a crucial feature! Hopefully there is some progress. Please update the community!
Thanks in advance!

**Comment by Filbuntu** [comment]
Hello @Astrid-Wire and @lucilacaranza!
It is still on top of my feature request list. It is almost two years since you wrote it is a "great feature request!" An alternative would be to "star" or mark in a different way a contact or message in order to know who needs an answer. The more communication we get the harder it is to not miss to answer.

**Comment by Filbuntu** [comment]
Hello @Astrid-Wire and @lucilacaranza !
I just realised that I am not the only one eagerly waiting for a similar feature for a long time. Here are other similar feature requests to link them together:
[Feature request: "Important messages" · Issue #78](https://github.com/wireapp/wire/issues/78) 
[pinned items, custom icon for chats that contain just one user · Issue #203](https://github.com/wireapp/wire/issues/203)
Need two important features to be add · Issue #199 

[Telegram added a new version](https://www.xda-developers.com/telegram-update-multiple-pinned-messages-distance-alerts-live-locations/) of pinned messages. Recently it is getting harder to convince friends to join Wire as Signal and Telegram have fast and positive developments  and new helpful features.

Please work on this, it would make Wire much more usable and efficient!

**Comment by ZelphirKaltstahl** [comment]
One point that can be made in favor of Wire is, that Signal requires you to have the app installed on phone, otherwise your account will not work or be removed (forgot which one) on computer and Telegram requires you to use a phone number.

Perhaps you can check out Deltachat, which only requires any e-mail address. However, it does not offer anything realtime, as it works using e-mails as a means of transporting messages.

**Comment by Filbuntu** [comment]
> One point that can be made in favor of Wire is, that Signal requires you to have the app installed on phone, otherwise your account will not work or be removed (forgot which one) on computer and Telegram requires you to use a phone number.

Yes, I agree Wire still has some feature I don't want to miss but my friends see it differently and Signal is catching up and may drop the need for a phone number in the near future.

> Perhaps you can check out Deltachat, which only requires any e-mail address. However, it does not offer anything realtime, as it works using e-mails as a means of transporting messages.

I love the concept of Deltachat, not needing to be part of one bubble/provider but the lack of audio and video calls is preventing me to join as my friends would not want an messaging app without it. There are some discussions going on (https://support.delta.chat/t/webrtc-for-communicating/410/25, https://support.delta.chat/t/video-chat-integration/388/22) to address this. Maybe they can integrate it sooner than later.

Nevertheless I hope I can stay with Wire and convince more friends to join but Wire **needs to continue to develop its system and apps**, maybe at a faster pace than in the past years. **And I hope this feature will be added very soon.**

**Comment by Filbuntu** [comment]
Hello @Astrid-Wire and @lucilacaranza!
Three long years this crucial feature request has been pending! A friend just told me yesterday that he does not like to use Wire because this feature is still missing. Could you please work on this?
Thanks in advance!

**Comment by Filbuntu** [comment]
Hello @Astrid-Wire  and @lucilacaranza!
I would appreciate a feedback, even a short one, please!
Thanks!

**Comment by matthiaskiller** [comment]
it's 2024 and this is still missing.
Could you please comment on the roadmap for this feature?
Cheers!

**Comment by davelowensohn** [comment]
Chiming in that I have to use Wire for multiple work projects and not having this feature makes it really hard to stay on top of things – when something comes in that needs a reply I'm faced with the choice of (1) reading and replying as it comes in, which I'm not always equipped to do (eg, if it's a document request), (2) reading the message and logging elsewhere that I need to get back to the person when I have what they need, which is cumbersome, or (3) leaving the message unread (like a coward!!) and gleaning what I can from the preview, so I'm able to see that I have an unread message to get back to when I'm back at my workstation and able to get the requester what they need.

I know "mark unread" isn't the greatest project management style, but it's one that has worked for me and my various collaborators for decades in Gmail, Slack, Discord, Signal, Apple Messages, the various apps I use for my kids' schools, etc etc. It's a feature that would make this app loads better for professional work & I just wanted to add my voice to the chorus of people requesting it. Thanks!
